apisix/plugins/authz-keycloak.lua

```diff
@@ -722,12 +722,12 @@ local function generate_token_using_password_grant(conf,ctx)
     if not username then
         local err = "username is missing."
         log.error(err)
-        return 422, err
+        return 422, {message = err}
     end
     if not password then
         local err = "password is missing."
         log.error(err)
```

Comment:
Shall we use log level `warn` instead of `error` level with `4xx` status code ?

Reply:
fixed